Changelog.md

```diff
@@ -46,6 +45,10 @@
 - Voice: Enable worldmap voice acting
 - Voice: Enable tutorial voice acting
 
+## FF7 Steam
+
+- Common: Fix a softlock when trying to change in-game Controller setting to "Normal" (by disabling this option)
+
```

Comment:
Thanks! Sorry maybe it was not clear on the previous comment, but can you please put this section after FF7? :)